scripts/benchmarks/benchmark.py

```diff
@@ -430,8 +428,8 @@ def is_completed(self):
         # return complete only if no errors
         # and is not a dry run
```

Comment:
lets not format anything in `scripts/benchmarks` since there is another PR #69 making changes to these files.